astroplan/scheduling.py

```diff
@@ -277,36 +439,35 @@ def _make_schedule(self, blocks):
 
             if block_constraint_results[bestblock_idx] == 0.:
                 # if even the best is unobservable, we need a gap
-                new_blocks.append(TransitionBlock({'nothing_observable': self.gap_time},
-                                                  current_time))
                 current_time += self.gap_time
             else:
                 # If there's a best one that's observable, first get its transition
                 trans = block_transitions.pop(bestblock_idx)
                 if trans is not None:
-                    new_blocks.append(trans)
+                    self.schedule.insert_slot(trans.start_time, trans)
                     current_time += trans.duration
 
                 # now assign the block itself times and add it to the schedule
                 newb = blocks.pop(bestblock_idx)
                 newb.start_time = current_time
-                current_time += self.gap_time
+                current_time += newb.duration
                 newb.end_time = current_time
                 newb.constraints_value = block_constraint_results[bestblock_idx]
 
-                new_blocks.append(newb)
+                self.schedule.insert_slot(newb.start_time, newb)
 
-        return new_blocks, True
+        return self.schedule
 
 
 class PriorityScheduler(Scheduler):
     """
     A scheduler that optimizes a prioritized list.  That is, it
     finds the best time for each ObservingBlock, in order of priority.
     """
+
     @u.quantity_input(gap_time=u.second)
     def __init__(self, start_time, end_time, constraints, observer,
-                 transitioner=None, gap_time=30*u.min, slew_time=5*u.min):
+                 transitioner=None, gap_time=30 * u.min, slew_time=5 * u.min):
```

Comment:
I realise this is a hangup from someone else's code, but is the ```slew_time``` a stop-gap until we have working ```Transitioners```? I don't think we need both. If we do need both for some reason, it's probably  more useful to supply the slew *rate* and work out the slew time once the from-to coordinates are known.